Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -190,7 +161,7 @@ struct HTTPHeadersIterator: IteratorProtocol {
 /// field when needed. It also supports recomposing headers to a maximally joined
 /// or split representation, such that header fields that are able to be repeated
 /// can be represented appropriately.
-public struct HTTPHeaders: Sequence, CustomStringConvertible {
+public struct HTTPHeaders: CustomStringConvertible {
```

Comment:
@Lukasa I may be wrong but isn't this a breaking change and so we need a next major version ?

Reply:
ah never mind its just moved.

Reply:
That's just breaking the conformance out into a separate extension. This is only a style thing but it's generally considered better than conforming in the struct's declaration (this was one of the motivating reasons to make `private` allow access from same-file extensions.